doc/classes/ItemList.xml

```diff
@@ -386,26 +399,35 @@
 	</methods>
 	<members>
 		<member name="allow_reselect" type="bool" setter="set_allow_reselect" getter="get_allow_reselect">
-			If [code]true[/code], the currently selected item may be selected again.
+			If [code]allow_reselect[/code] is set to true, the currently selected item can be selected again.
 		</member>
 		<member name="allow_rmb_select" type="bool" setter="set_allow_rmb_select" getter="get_allow_rmb_select">
-			If [code]true[/code], a right mouse button click can select items.
+			If [code]allow_rmb_select[/code] is set to true, right mouse button click can select items.
 		</member>
 		<member name="auto_height" type="bool" setter="set_auto_height" getter="has_auto_height">
+			If [code]auto_height[/code] is set to [code]true[/code], the control will automatically resize the height to fit its content.
 		</member>
 		<member name="fixed_column_width" type="int" setter="set_fixed_column_width" getter="get_fixed_column_width">
+			Sets the default column width in pixels.
+			if [code]fixed_column_width[/code] is set to default [code]0[/code] then each item will have a width equal to the width of its content and the columns will have an uneven width.
 		</member>
 		<member name="fixed_icon_size" type="Vector2" setter="set_fixed_icon_size" getter="get_fixed_icon_size">
+			Sets the default icon size in pixels.
 		</member>
 		<member name="icon_mode" type="int" setter="set_icon_mode" getter="get_icon_mode" enum="ItemList.IconMode">
+			Sets the position of the icon to either [code]top[/code] or [code]left[/code].
```

Comment:
"top" and "left" are not actual values that can be set for this property. It would be better to refer to `[const ICON_MODE_TOP]` and `[const ICON_MODE_LEFT]`.